### DISCUSSION TITLE
Parakeet model support on RIVA

### DISCUSSION THREAD

**Discussion by lakshmi-speak** [NVIDIA-NeMo/NeMo#8190] [General]
Are there plans to support the [Parkeet](https://nvidia.github.io/NeMo/blogs/2024/2024-01-parakeet/) family of models on the RIVA Platform for streaming inference?

**Comment by arunvenkatesan-nv** [comment]
Yes, we plan to add support for the Parakeet series of models in Riva over the next few releases. To start with, we expect to have Parakeet-ctc-0.6b trained on better curated data available with the upcoming Riva release in February. We expect this model to significantly improve accuracy on real world test sets.

**Reply by lakshmi-speak** [reply]
Great to hear! Will it support streaming?

**Reply by arunvenkatesan-nv** [reply]
Yes, it will support streaming

**Reply by rodrigoGA** [reply]
Do you already have a release date? It would be interesting if a model for Spanish is also incorporated

**Reply by francois-vz** [reply]
A release date would be great if at all possible @arunvenkatesan-nv

**Reply by martinkallstrom** [reply]
Any updates on plans for streaming parakeet?

**Reply by rodrigoGA** [reply]
I'm also waiting; it would be nice if with the paid version of riva they at least gave access to a serious roadmap, so we can know what the internal team can work on and what to expect from Riva. Because from the news in the blogs, they often publish many articles saying that they will be added to the next version of Riva, but that's all there is to it.

**Reply by anshulwadhawan** [reply]
I see support for the ctc model here: https://docs.nvidia.com/ai-enterprise/nim-asr/latest/parakeet-ctc-riva.html. Is there any timeline for adding support for parakeet tdt ctc streaming?
